packages/traceability/issuer-irec-api-wrapper/src/Issue.ts

```diff
@@ -65,7 +65,21 @@ export class ApproveIssue {
 }
 
 export class IssueWithStatus extends Issue {
+    @IsString()
+    @IsNotEmpty()
     code: string;
 
-    status: IssueStatus;
+    @IsNotEmpty()
+    @IsString()
+    @IsIn(Object.values(IssueStatus))
+    status:
+        | IssueStatus.Approved
+        | IssueStatus.Draft
+        | IssueStatus.InProgress
+        | IssueStatus.Rejected
+        | IssueStatus.Referred
+        | IssueStatus.Verified
+        | IssueStatus.Issued
+        | IssueStatus.Submitted
+        | IssueStatus.Withdrawn;
```

Comment:
Would this not work fine?

```js
@IsNotEmpty()
@IsEnum(IssueStatus)
status: IssueStatus;
```

Reply:
But `IssueStatus` sounds like a "status of a problem (issue)" to me.
`IssuanceStatus` would make it more clear IMO.

Reply:
Agree, fixed it

Reply:
Oops, looks like I pushed this commit to the wrong branch. Now it is here